metadata-ws/src/test/resources/features/security.feature

```diff
@@ -0,0 +1,82 @@
+Feature: Security related tests
+
+  Scenario: access security disabled links
+    When I request unauthorized GET /
+    Then the response code should be 200
+    When I request unauthorized GET /swagger-ui.html
+    Then the response code should be 200
+
+
+  Scenario: update security enabled links
+    When I request POST /publications with JSON payload:
+    """
+    {
+      "publicationId": "Publication5"
+    }
+    """
+    And set the URL to PUBLICATION5
+    Then the response code should be 201
+
+    When I request PUT PUBLICATION5 with JSON payload:
+    """
+    {
+      "publicationId": "Publication7"
+    }
+    """
+    Then the response code should be 204
+    When I request GET with value of PUBLICATION5
+    Then the response code should be 200
+    And the response should contain field publicationId with value Publication7
+
+    When I request DELETE with value of PUBLICATION5
+    Then the response code should be 204
+
+
+  Scenario: access security enabled links and fail
+    When I request unauthorized POST /publications with JSON payload:
```

Comment:
instead of using a generic reference like unauthorized, use a "Given " with a user and some roles and try the request

Reply:
same for all cases

Reply:
Pls see above comment

Reply:
The use case is for Security and roles so it would be meaningful if we have Given/When with the user and his roles not just authorized/unauthorized.

Reply:
@selvabi If you're happy with the changes, let's mark this as resolve